client/src/styles/AdvancedWorkflowAdmin.scss

```diff
@@ -15,11 +15,11 @@
 .cms-tree.jstree {
   span.badge.status-embargo,
   span.badge.status-expiry,
-  span.badge.status-embargo_expiry {
+  span.badge.status-embargo_expiry {// sass-lint:disable-line class-name-format
```

Comment:
I would be fined if we updated those classes in the actual HTML so we don't have to use bad call names and ids in the CSS.

Reply:
I leave it as is?!